ale/drivers/lro_drivers.py

```diff
@@ -222,6 +240,34 @@ def sampling_factor(self):
         """
         return self.crosstrack_summing
 
+    @property
+    def spacecraft_direction(self):
+        """
+        Returns the x axis of the first velocity vector relative to the
+        spacecraft. This indicates of the craft is moving forwards or backwards.
+
+        From LROC Frame Kernel: lro_frames_2014049_v01.tf
+        "+X axis is in the direction of the velocity vector half the year. The
+        other half of the year, the +X axis is opposite the velocity vector"
+
+        Hence we rotate the first velocity vector into the sensor reference
+        frame, but the X component of that vector is inverted compared to the
+        spacecraft so a +X indicates backwards and -X indicates forwards
```

Comment:
This is only true for the NACL reference frame, has this been tested on both left and right images?